test/unit/wallet/nep2.js

```diff
@@ -2,13 +2,21 @@ import * as NEP2 from '../../../src/wallet/nep2'
 import { isNEP2, isWIF } from '../../../src/wallet/verify'
 import testKeys from '../testKeys.json'
 
-describe('NEP2', function () {
+require('babel-polyfill')
+
+describe.only('NEP2', function () {
   const simpleScrypt = {
     cost: 256,
     blockSize: 1,
     parallel: 1,
     size: 64
   }
+  const simpleAsyncScrypt = {

```

Comment:
we should follow the old scrypt conventions. Or change everything to n,r,p . `dkLen` is not variable (fixed at 64)

Reply:
ok, will make these changes later. and I will follow the old conventions